docs/qa/method.md

```diff
@@ -29,7 +29,7 @@ _200 Node Test_, and _Rotating Nodes Test_.
     * [Terraform CLI][Terraform]
     * [Ansible CLI][Ansible]
 
-[testnet-repo]: https://github.com/interchainio/tendermint-testnet
+[testnet-repo]: https://github.com/interchainio/tendermint-testnet   FIX
```

Comment:
Is this  ToDo? Or we should leave it in to remember to fix it?

Reply:
Wow, didn't even see it :-(
Bear with me while I fix it